Pyrado/pyrado/sampling/step_sequence.py

```diff
@@ -343,6 +343,12 @@ def __init__(
         else:
             self._rollout_bounds = None
 
+    def __repr__(self) -> str:
+        return f"StepSequence[rewards={self.rewards}, observations={self.observations}, actions={self.actions}]"
+
+    def __str__(self) -> str:
+        return str(self.observations)
```

Comment:
Is this used for the hash?

Reply:
No, I used it for debugging the smoke test to check whether the sampled sequences are different.